homeassistant/components/tuya/number.py

```diff
@@ -307,6 +307,17 @@
             icon="mdi:thermometer-lines",
         ),
     ),
+    # Smart Water Timer
+    # UNDOCUMENTED
+    "sfkzq": (
+        NumberEntityDescription(
```

Comment:
Does this entity count down while running?

Reply:
Yes it does. It's set, then counts down once switch is set to true. Should this be setup elsewhere?

Reply:
We down allow number entities to be used for countdowns.

../Frenck